src/functionalTest/java/submitBulk/ClaimStatusUpdate.java

```diff
@@ -0,0 +1,65 @@
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
```

Comment:
These are JUnit4 classes.  Replace them with the JUnit5 equivalents:
```
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;
```
More details about replacing `@RunWith` with `@ExtendWith` can be found in the comment for that line.